### PR TITLE
Cache locations via booking locations gem

### PR DESCRIPTION
**Pull request by benlovell**
Location caching is now provided (and configurable) by the booking
locations gem since v0.6. We should use that instead of the customised
caching code in this project.

### CODE REVIEW

File: config/initializers/online_booking_locations.rb [hunk 1]
```diff
@@ -13,11 +13,9 @@
...
-                         BookingLocations::StubApi.new
-                       else
-                         require 'cached_booking_locations_api'
-                         CachedBookingLocationsApi.new
-                       end
+if Rails.env.development?
+  require 'booking_locations/stub_api'
+  BookingLocations.api = BookingLocations::StubApi.new
+else
+  BookingLocations.cache = Rails.cache
```
Comment (benbarnett) [inline config/initializers/online_booking_locations.rb]: Do we need to consider the test environment here? Or does that just get bypassed elsewhere?

Comment (benlovell) [inline config/initializers/online_booking_locations.rb]: Test environment does the right thing explicitly, depending on the needs of the particular test. 👍

**Comment by benlovell** [comment]
@benbarnett is this ok to merge?

**Comment by benbarnett** [comment]
Yep 👍

**Pull request merged**